src/_posts/databases/redis/2000-01-01-start.md

```diff
@@ -270,6 +270,15 @@ Beware that no downgrade is possible once your database has been upgraded.
   </table>
 </div>
 
+### Database Users
```

Comment:
Nitpick: I wonder if it wouldn't more relevant to place this section under the `Redis Configuration` one.
Because I think it is more related to the configuration of the database instead of the dashboard since it is not possible to create a new user from it.

Reply:
I wanted this section to be at the exact same place on all database types doc pages. That's why I wrote it here. But if you think it should be moved I can. What do you prefer? :)